.gitignore

```diff
@@ -0,0 +1,2 @@
+.pytest_cache
+*.egg-info
```

Comment:
you need __pycache__ in here too. You accidentally committed a __pycache__file in the prior commit.